Sources/TokamakCore/Modifiers/HoverActionModifier.swift

```diff
@@ -0,0 +1,36 @@
+// Copyright 2021 Tokamak contributors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+protocol HoverActionType {
```

Comment:
Is there any purpose to this protocol? It only seems to be used in the extension below, but doesn't seem to be required there.

Reply:
Good catch. I was replicating `AppearanceActionType`, but that protocol is actually used elsewhere for casting, justifying its existence.